metals/src/main/scala/scala/meta/internal/metals/StacktraceAnalyzer.scala

```diff
@@ -125,6 +125,7 @@ class StacktraceAnalyzer(
   private def makeGotoCommandParams(
       location: Location
   ): l.ExecuteCommandParams = {
+    // TODO
```

Comment:
Need to figure out what to do about `GotoLocation` since it doesn't fit into the single param commands.